src/job/job.service.ts

```diff
@@ -54,6 +56,33 @@ export class JobService {
     };
   }
 
+  async createJobsByEvent(dto: CreateClaimItemsEventDto) {
+    const targetTickers = dto.items.map((item) => item.ticker);
```

Comment:
```suggestion
    const targetTickers = [...new Set(dto.items.map((item) => item.ticker))];
```

Reply:
Though the JavaScript `Set` doesn't work for object types, it will be okay because the type is `Set<string>`. 👍🏻